nixos/modules/programs/qgroundcontrol.nix

```diff
@@ -0,0 +1,39 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.qgroundcontrol;
+in
+{
+
+  options = {
+    programs.qgroundcontrol = {
+      enable = lib.mkEnableOption "qgroundcontrol";
+
+      package = lib.mkPackageOption pkgs "qgroundcontrol" {};
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+
+    # ModemManager is known to interfere with serial connections;
+    # qgroundcontrol recommends disabling it
+    systemd.services.ModemManager.enable = lib.mkDefault false;
+
+    # Security wrapper
+    security.wrappers.qgroundcontrol = {
+      source = "${pkgs.qgroundcontrol}/bin/qgroundcontrol";
+      owner = "root"; # Sensible default; not setuid so this is not a security risk
+      group = "tty";
+      setuid = false;
```

Comment:
This is `null` by default, so it's probably not needed. The comment above already asserts it, so there shouldn't be any confusion either.

```suggestion
```